robot/EDA/tests/browser/settings/contact_information/preferred_email_phone.robot

```diff
@@ -0,0 +1,46 @@
+*** Settings ***
+Documentation   Validates preferred email and phone in contact information settings
+Resource        robot/EDA/resources/EDA.robot
+Library         cumulusci.robotframework.PageObjects
+Suite Setup     Open Test Browser
+Suite Teardown  Run Keywords
+...             Update settings to out of the box values        AND
+...             Capture screenshot and delete records and close browser
+
+Test Setup      Go to education cloud settings
+*** Keywords ***
+Update settings to out of the box values
+    Go to education cloud settings
```

Comment:
This is part of the `Test Setup`, do you need to add it again in the test case?

Reply:
Yes. I still have few more tests to be added to this contact information settings.